docs/installation/HelmChart.md

```diff
@@ -0,0 +1,10 @@
+# Kubernetes - Helm chart
+
+We provide [Helm chart](https://artifacthub.io/packages/helm/touk/nussknacker) with basic Nussknacker setup, including:
+
+- Kafka - required only in streaming processing mode
+- Grafana + InfluxDB
+- One of the available engines: Flink or Lite.
+
+Please note that Kafka (and Flink if chosen) are installed in basic configuration - for serious production deployments you probably
```

Comment:
I think `serious production deployments` doesn't really add much, i'd remove it. As far as I understand it, essentially what needs to be said is that if you want to configure services you can configure them through the helm chart.